introduction_to_amazon_algorithms/xgboost_abalone/xgboost_parquet_input_training.ipynb

```diff
@@ -21,6 +21,7 @@
     "import os\n",
```

Comment:
<p>Can we change this to <code>python -m pip install</code>  instead? I've encountered problems with the <code>pip</code>  CLI wrapper when I changed notebook kernels. See <a href="https://github.com/pypa/pip/issues/3164#issuecomment-398216682" target="_blank">https://github.com/pypa/pip/issues/3164#issuecomment-398216682</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>